### ISSUE TITLE
Hide functions and widgets in adjacent modules

### ISSUE THREAD

**Issue by Joshdpaul** [ua-snap/ardac-notebooks#14]
For each notebook, can the functions & widget be "hidden" in an adjacent module, so that the user can just plug in a lat long, shift+enter, and boom heres the widget?

The intention would be to hide the code from people who won't really want to see it, and bring the user input > widget functionality front and center. 

Depends on the audience - are these notebooks for people who would be thrilled to see all the code ASAP, or more thrilled to see the widget ASAP? Also, depends on the tech; @charparr mentioned this might be difficult "because pyodide and web assembly (importing local modules doesn’t work well, and requests and imports are weird)"

**Comment by charparr** [comment]
Yeah, this was the original design! But the import situation with the pyodide kernel is not great. An alternative kernel, like [xeus python](https://github.com/jupyter-xeus/xeus-python) might handle this better. I'm keeping this issue open, but for the immediate future users will get a blap of all code right there in the notebook.
